syncd/SwitchNotifications.h

```diff
@@ -118,6 +118,7 @@ namespace syncd
                             .on_switch_asic_sdk_health_event = &Slot<context>::onSwitchAsicSdkHealthEvent,
                             .on_port_host_tx_ready = &Slot<context>::onPortHostTxReady,
                             .on_twamp_session_event = &Slot<context>::onTwampSessionEvent,
+                            .on_icmp_echo_session_state_change = nullptr,
```

Comment:
Any other changes required due to SAI refpoint update?

Reply:
> Any other changes required due to SAI refpoint update?

No. Later let icmp echo feature owner implement its detail notification callback.